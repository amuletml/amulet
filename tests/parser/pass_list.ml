let _ = [1,2,3]
let [1,2,3] = [1,2,3]
let _ = [ let a = 1 in 2 ]

let _ = [ (a, b) | with a <- [1, 2, 3], with b <- [ 4, 5, 6 ] ]
let _ = [ (a, b) | with a <- [ 1, 2, 3 ], let b = a + 1, with c <- [ b, a ] ]
