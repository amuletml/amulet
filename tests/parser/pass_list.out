let _ = [1, 2, 3]
let [1, 2, 3] = [1, 2, 3]
let _ = [let a = 1
  in 2]
let _ =
  [(a, b) | a <- [1, 2, 3], b <- [4, 5, 6]]
let _ =
  [(a, b) | a <- [1, 2, 3], let b =
    (a + 1) , c <- [b, a]]
